.gitignore

```diff
@@ -1,17 +1,20 @@
 node_modules
 *.log
 .idea
+
+package-lock.json
```

Comment:
@tmeasday see my PR here: https://github.com/storybooks/storybook/pull/1338

We tried merging in the package lock, but it broke everything, and I pulled it out. I think until somebody figures out what to do here, we are better off not checking them in and ignoring them so that they don't accidentally get checked in.